lib/src/collections/observable_set.dart

```diff
@@ -116,3 +133,86 @@ class _DelegatingObservableSet<E> extends DelegatingSet<E>
     removeWhere((e) => !test(e));
   }
 }
+
+class _UnmodifiableEmptySet<E> extends IterableBase<E> implements Set<E> {
```

Comment:
I will file a bug against the core-lib. As soon as it supports `const Set()`, I can remove this.